Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM rust:1.34.0
+ARG RUSTC_VERSION
+FROM rust:${RUSTC_VERSION}
 
 RUN cargo install cargo-watch
 RUN apt-get update && apt-get install -y ruby
```

Comment:
Unrelated to this PR, and probably would fit better in another PR: This Dockerfile is building as root, which [isn't recommended](http://canihaznonprivilegedcontainers.info). You can do `useradd --create-home myusernamehere` followed by `USER myusernamehere` to drop privileges; [here's an example](https://pythonspeed.com/articles/dockerizing-python-is-hard/).

Reply:
Hmm, building as non-root is a bit ugly, because of this issue: https://github.com/moby/moby/issues/6119. Building as root and than just **running** the executable as non-root would be easier. What exactly are the security concerns?